pay-api/migrations/versions/2025_07_04_a7b9c2d4e6f8_add_ob_credits_and_pad_credits_to_payment_accounts.py

```diff
@@ -0,0 +1,33 @@
+"""add ob_credit and pad_credit to payment_accounts
+
+Revision ID: a7b9c2d4e6f8
+Revises: afccd441770e
+Create Date: 2025-01-30 10:00:00.000000
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+
+# revision identifiers, used by Alembic.
+revision = 'a7b9c2d4e6f8'
+down_revision = 'b927265cf3a1'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    """Add ob_credit and pad_credit columns to payment_accounts table and drop credit column."""
+    op.add_column('payment_accounts', sa.Column('ob_credit', sa.Numeric(19, 2), nullable=True))
+    op.add_column('payment_accounts', sa.Column('pad_credit', sa.Numeric(19, 2), nullable=True))
+    op.drop_column('payment_accounts', 'credit')
```

Comment:
how are we handling the payment_accounts if credit currently has values？

Reply:
![Uploading image.png…]()